test/edu-benefits/5490/config/educationHistory.unit.spec.jsx

```diff
@@ -58,7 +58,7 @@ describe('Edu 5490 educationHistory', () => {
       }
     });
 
-    expect(formDOM.querySelectorAll('input,select').length).to.equal(12);
+    expect(formDOM.querySelectorAll('input,select').length).to.equal(10);
```

Comment:
Can you add a test that verifies that when you choose `expected` for the high school status you get the diploma date and not the trainings question? Should look pretty similar to the `should render diploma date` test.